src/index.ts

```diff
@@ -584,9 +585,15 @@ function createServicePrincipal(azureWebsiteName: string,
         return createAppRoleAssignment(adminAppRoleId, sp, graphClient, baseUri);
     })
     .then((sp: any) => {
+        options.tokenAudience = undefined;
```

Comment:
Wondering if we can move the entire createServicePrincipal in the AzureHelper. BTW, I like the clean separate with AzureHelper!!